### PR TITLE
Add apimodules to test filter

### PR DESCRIPTION
**Pull request by reutshal**
## Status
- [X] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/23141

## Description
A few sentences describing the overall goals of the pull request's commits.

### CODE REVIEW

**Comment by reutshal** [comment]
@ronykoz This is an example of a build in which I changed the CSVFeedApiModule , you can see under test-filter the integrations that was added: https://circleci.com/gh/demisto/content/54339#build-parameters/containers/0

File: dev-requirements-py3.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 flake8==3.7.9
 bandit==1.6.2
 demisto-py==2.0.11
-demisto-sdk==1.0.8
+git+https://github.com/demisto/demisto-sdk.git@add-apiModules_to_id_set
```
Comment (yaakovi) [inline dev-requirements-py3.txt:4]: don't forget to revert

**Comment by reutshal** [comment]
@ronykoz Tests that were made:
* Added a new ApiModule + imported in integration and checked it appears in the tests.
* modified an ApiModule and checked all relevant integrations appears in the tests.

File: Tests/scripts/collect_tests_and_content_packs.py [hunk 2]
```diff
@@ -544,11 +544,25 @@ def collect_content_packs_to_install(id_set: Dict, integration_ids: set, playboo
     return packs_to_install
 
 
+def get_api_module_integrations(changed_api_modules, integration_set):
+    integration_to_version = {}
+    integration_ids_to_test = set([])
+    for integration in integration_set:
+        if list(integration.values())[0].get('api_modules', '') in changed_api_modules:
```
Comment (ronykoz) [inline Tests/scripts/collect_tests_and_content_packs.py:551]: let's save this one as arg `list(integration.values())[0]`

**Comment by yaakovi** [comment]
@reutshal unit tests are failing

**Pull request merged**